components/subreddits/subHeader.tsx

```diff
@@ -12,12 +12,12 @@ const SubHeader = ({ setter, url }) => {
     <div className={styles.container}>
       <div className={styles.info}>
         <h1 data-testid="sub-title">
-          {data.data ? data.data.display_name_prefixed : data.error}
+          {data?.data ? data.data.display_name_prefixed : '404'}
```

Comment:
Is this is a hack around the undefined data object? What would the professional do?